uiowa/uiowa
# Try Replicate and Replicate UI modules

Related to #2848 
Related to #4223 

# How to test



- Checkout branch and composer install
- Sync a site and uli
- From a node page or article, or `admin/content`, choose to "replicate" -- enter a new title
- Check that the page has been successfully replicated
  - All content should be present on the new page
  - New blocks should be created
  - New paragraphs nested under blocks should be created
  - Section components should have new UUIDs
  - New copies of referenced nodes (article authors, for instance), media, taxonomy terms, etc. should not be created, but the references should still be there
  - Check the revision overview tab--there should be a single revision, with a message that it was replicated from another node (with a link to that node).
- Only Page and Article content types should be replicable. For other content types, you should not have the option to replicate.
  - If you manually enter "node/#/replicate" for a non-replicable page, you should not be allowed to replicate, and should see a message stating this
- Publisher and webmaster should have replicate permissions. Related config splits that needed updating can be checked from https://github.com/uiowa/uiowa/search?p=1&q=filename%3Auser.role.webmaster or https://github.com/uiowa/uiowa/search?q=filename%3Auser.role.publisher
- Sitenow V2 sites should not have the option to replicate
